src/exchange/swap/getExchangeRates.ts

```diff
@@ -16,16 +16,19 @@ import {
 const getExchangeRates: GetExchangeRates = async (
   exchange: Exchange,
   transaction: Transaction,
-  userId?: string // TODO remove when wyre doesn't require this for rates
+  userId?: string, // TODO remove when wyre doesn't require this for rates
```

Comment:
this is no longer required for Wyre to give us the rates, you can verify (connected to a US vpn)
```
curl -L -X POST 'https://swap.ledger.com/v3/rate' \
-H 'Content-Type: application/json' \
--data-raw '{"from": "bitcoin", "to": "ethereum", "amountFrom": "1"}'
```

Reply:
I'll double check 👍